test/src/main/resources/resources.yml

```diff
@@ -0,0 +1,110 @@
+#    type: "oracle.jdbc.xa.client.OracleXADataSource"
+#    url: "jdbc:oracle:thin:@${jdbc.hostname:-localhost}:1521:XE"
+#    authalias: "${db.authalias}"
+#    username: "testiaf_user"
+#    password: "testiaf_user00"
+
+
+#  - name: "ibis4test-mssql"
+#    type: "com.microsoft.sqlserver.jdbc.SQLServerXADataSource"
+#    url: "jdbc:sqlserver://${jdbc.hostname:-localhost}:1433;database=testiaf"
+#    authalias: "${db.authalias}"
+#    username: "testiaf_user"
+#    password: "testiaf_user00"
+#    properties:
+#      sendStringParametersAsUnicode: "false"
+#      sendTimeAsDatetime: "true"
+#      selectMethod: "direct"
+#      responseBuffering: "adaptive"
+#      packetSize: "4096"
+#      loginTimeout: "10"
+#      lockTimeout: "2000"
+#      lastUpdateCount: "false"
+#      encrypt: "false"
+#      disableStatementPooling: "true"
+#      applicationName: "iaf-test on TomCat"
+#      applicationIntent: "readwrite"
+
+#  - name: "ibis4test-mysql"
+#    type: "com.mysql.cj.jdbc.MysqlXADataSource"
+#    url: "jdbc:mysql://${jdbc.hostname:-localhost}:3307/testiaf"
+#    authalias: "${db.authalias}"
+#    username: "testiaf_user"
+#    password: "testiaf_user00"
+#    properties:
+#      sslMode: "DISABLED"
+#      serverTimezone: "Europe/Amsterdam"
+#      allowPublicKeyRetrieval: "true"
+#      pinGlobalTxToPhysicalConnection: "true"
+#      socketTimeout: "5000"
+
+
+#  - name: "ibis4test-mariadb"
+#    type: "org.mariadb.jdbc.MariaDbDataSource"
+#    url: "jdbc:mariadb://${jdbc.hostname:-localhost}:3306/testiaf"
+#    authalias: "${db.authalias}"
+#    username: "testiaf_user"
+#    password: "testiaf_user00"
+
+    # Postgres XA does not work without Narayana, gives ClassCastException casting the XA datasource to Java datasource, so add non-XA for those cases
+  - name: "ibis4test-postgres"
+    type: "org.postgresql.Driver"
+    url: "jdbc:postgresql://${jdbc.hostname:-localhost}:5432/testiaf"
+    authalias: "${db.authalias}"
+    username: "testiaf_user"
+    password: "testiaf_user00"
+
+#  - name: "ibis4test-postgres-xa"
+#    type: "org.postgresql.xa.PGXADataSource"
+#    url: "jdbc:postgresql://${jdbc.hostname:-localhost}:5432/testiaf"
+#    authalias: "${db.authalias}"
+#    username: "testiaf_user"
+#    password: "testiaf_user00"
+
+  - name: "ibis4test-db2"
+    type: "com.ibm.db2.jcc.DB2SimpleDataSource"
+    url: "jdbc:db2://${jdbc.hostname:-localhost}:50000/testiaf"
+    authalias: "${db.authalias}"
+    username: "testiaf_user"
+    password: "testiaf_user00"
+
+#  - name: "ibis4test-db2-xa"
+#    type: "com.ibm.db2.jcc.DB2XADataSource"
+#    url: "jdbc:db2://${jdbc.hostname:-localhost}:50000/testiaf"
+#    authalias: "${db.authalias}"
+#    username: "testiaf_user"
+#    password: "testiaf_user00"
+
+mongodb:
+  - name: "ibis4test"
+    type: "com.mongodb.MongoClient"
+    url: "mongodb://${jdbc.hostname:-localhost}"
+    authalias: "${db.authalias}"
+    username: "testiaf_user"
+    password: "testiaf_user00"
+
+jms:
```

Comment:
Heb je dit wel getest met JMS? Ik zie dat JMS nu alles uitgecommentariseerd heeft.

Reply:
Ja, maar alleen met Tomcat, JBOSS gebruikt zijn eigen resource definities (thans dat is het gene dat we willen testen).